mmv1/third_party/terraform/fwprovider/framework_provider_terraform_attribution_label_addition_strategy_test.go

```diff
@@ -0,0 +1,98 @@
+package fwprovider_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+// TestAccFwProvider_terraform_attribution_label_addition_strategy is a series of acc tests asserting how the plugin-framework provider handles terraform_attribution_label_addition_strategy arguments
+// It is plugin-framework specific because the HCL used provisions plugin-framework-implemented resources
+// It is a counterpart to TestAccSdkProvider_terraform_attribution_label_addition_strategy
+func TestAccFwProvider_terraform_attribution_label_addition_strategy(t *testing.T) {
+	testCases := map[string]func(t *testing.T){
+		// Configuring the provider using inputs
+		"config sets terraform_attribution_label_addition_strategy values":                                                                                 testAccFwProvider_terraform_attribution_label_addition_strategy_configUsed,
+		"when terraform_attribution_label_addition_strategy is unset in the config, the default value'CREATION_ONLY' is NOT set in the provider meta data": testAccFwProvider_terraform_attribution_label_addition_strategy_defaultValue,
```

Comment:
Wouldn't we want the default value here? From what I understand `CREATION_ONLY` should be behavior that we would want to transfer to framework resources

Reply:
Does this mean that we need to explicitly set the default value somewhere?

Reply:
From a quick look, I think that [only the SDK logic currently handles the two attribution-related arguments](https://github.com/hashicorp/terraform-provider-google/blob/738e5b8246db5495fa66f8fd46f3ff59b133bde5/google/provider/provider.go#L924-L935). For example if I do a global search for `transport_tpg.CreateOnlyAttributionStrategy` I cannot see it in the PF code.

However, once we start using the SDK configuration logic on the PF side of the provider this disparity should go away, I hope. Unless there are other bits of labels-related code that needs to be implemented on resources/data sources, but I'm not familiar with that work.